index.html

```diff
@@ -7,72 +7,63 @@
+
+  <script>
+    var html = document.documentElement;
+    html.className = html.className.replace("without-js", "with-js");
+  </script>
+
+  <script type="text/javascript" src="/jquery.min.js"></script>
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.24.0/moment-with-locales.min.js"></script>
+  <script type="text/javascript" src="/index.js"></script>
 
 </head>
 
 <body>
 
-  <div class="navbar navbar-fixed-top">
-    <div class="navbar-inner">
-      <div class="container">
-        <button type="button" class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-        </button>
-        <li class="dropdown brand">
-          <a class="dropdown-toggle" data-target="navbar-dropdown-menu" href="#" data-toggle="dropdown">
-            <img src="https://nixos.org/logo/nix-wiki.png" alt="NixOS" class="logo" /> NixOS Status <b
-              class="caret"></b>
-          </a>
-          <ul class="dropdown-menu" id="navbar-dropdown-menu">
-            <li><a href="https://nixos.org">NixOS.org</a></li>
-          </ul>
-        </li>
-
-        <div class="nav-collapse collapse">
-          <ul class="nav pull-left">
-            <li><a href="https://monitoring.nixos.org/grafana">Grafana</a></li>
-            <li><a href="https://monitoring.nixos.org/prometheus">Prometheus</a></li>
-          </ul>
-        </div>
-      </div>
+  <header>
+    <div>
+      <h1 class="-with-subtitle">
+        <a href="https://nixos.org">NixOS</a>
+        <div class="subtitle">Infrastructure Status</div>
+      </h1>
+      <nav style="display: none;">
+        <ul>
+          <li><a href="https://monitoring.nixos.org/grafana">Grafana</a></li>
+          <li><a href="https://monitoring.nixos.org/prometheus">Prometheus</a></li>
+        </ul>
+      </nav>
     </div>
-  </div>
+    <button class="menu-toggle">Toggle the menu</button>
+  </header>
 
-  <div class="container main">
-    <div class="nixos-org-index">
-      <h1>NixOS</h1>
-    </div>
+  <div class="page-title">
+    <h1>Channel status</h1>
+  </div>
 
-    <div id="alerts"></div>
+  <div class="page-description">
+    <p>See the <a href="https://nixos.wiki/wiki/Nix_channels">NixOS Wiki</a>
+      for information on how channel updates progress.
+    </p>
+  </div>
 
-    <div class="hero12">
-      <h2>Nix Channel Status</h2>
-      <p>See the <a href="https://nixos.wiki/wiki/Nix_channels">NixOS Wiki</a>
-        for information on how channel updates progress.
-      </p>
+  <main>
+    <div id="alerts">
     </div>
 
-    <div class="hero12">
+    <div id="content">
       <noscript>
         <p class="jsfallback">This page requires Javascript.</p>
       </noscript>
 
-      <table class="table table-striped">
+      <table class="styled-table">
```

Comment:
```diff
diff --git a/index.html b/index.html
index 8f0a044..7e12ad8 100644
--- a/index.html
+++ b/index.html
@@ -63,18 +63,20 @@
         <p class="jsfallback">This page requires Javascript.</p>
       </noscript>
 
-      <table class="styled-table">
-        <thead>
-          <tr>
-            <th>Channel</th>
-            <th>Last updated</th>
-            <th>Commit</th>
-            <th>Hydra job for tests</th>
-            <th>Channel Status</th>
-          </tr>
-        </thead>
-        <tbody id="channel-status"></tbody>
-      </table>
+      <div class="responsivescroll-table">
+        <table class="styled-table">
+          <thead>
+            <tr>
+              <th>Channel</th>
+              <th>Last updated</th>
+              <th>Commit</th>
+              <th>Hydra job for tests</th>
+              <th>Channel Status</th>
+            </tr>
+          </thead>
+          <tbody id="channel-status"></tbody>
+        </table>
+      </div>
     </div>
   </main>
```

(Can't suggest as it goes outside the diff span)

This will make only the table content scrollable.

It is okay to use the class names, or alternatively [the mixins could be used](https://github.com/samueldr/nixos-homepage/commit/ac8c47866c87a3dd75fb9bca163368552cfbf7ba#diff-a4634871f0349e8fc2010edbe1924cfd74868e6c858ecd3dfcfdd913719854deR153).